packages/frontend-api/src/Model/Mutation/Customer/DeliveryAddress/DeliveryAddressDataFactory.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Shopsys\FrontendApiBundle\Model\Mutation\Customer\DeliveryAddress;
+
+use Overblog\GraphQLBundle\Definition\Argument;
+use Shopsys\FrameworkBundle\Model\Country\CountryFacade;
+use Shopsys\FrameworkBundle\Model\Customer\Customer;
+use Shopsys\FrameworkBundle\Model\Customer\DeliveryAddressData;
+use Shopsys\FrameworkBundle\Model\Customer\DeliveryAddressDataFactory as FrameworkProductRepository;
+use Shopsys\FrameworkBundle\Model\Pricing\Exception\InvalidArgumentException;
+
+class DeliveryAddressDataFactory
+{
+    /**
+     * @param \Shopsys\FrameworkBundle\Model\Customer\DeliveryAddressDataFactory $frameworkProductRepository
+     * @param \Shopsys\FrameworkBundle\Model\Country\CountryFacade $countryFacade
+     */
+    public function __construct(
+        protected readonly FrameworkProductRepository $frameworkProductRepository,
```

Comment:
the alias (and the parameter) should probably be named `FrameworkDeliveryAddressDataFactory`.
But if you rename this class to `DeliveryAddressDataApiFactory`, there will be no problem :grin:

Reply:
Thx. I should not trust github copilot :grimacing: